collections/_article/godotcon-2023-cfp.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Godot Conference 2023: Call for Participations"
+excerpt: "GodotCon 2023 needs participations"
+categories: ["events"]
+author: Julian
+image: /storage/blog/covers/godotcon-2023.webp
+image_caption_description: Image by Raffaele Picca
+date: 2023-08-06 17:00:00
+---
+
+We are organizing the annual Godot Conference that will happen [lively at Munich](https://godotengine.org/article/godotcon-2023/) for the 
+first time since the last occurrence. We are really thrilled to meet again fellows Godot users and developers to share experiences during the 2 days of the event.
```

Comment:
```suggestion
first time since the last occurrence. We are really thrilled to meet again fellow Godot users and developers to share experiences during the 2 days of the event.
```